ui/main.qml

```diff
@@ -198,13 +198,13 @@ StatusWindow {
         Theme.changeTheme(localAppSettings.theme, systemPalette.isCurrentSystemThemeDark())
         Theme.changeFontSize(localAccountSensitiveSettings.fontSize)
 
-        d.runMockedKeycardControllerWindow()
+        d.runMockedKeycardControllerWindow() // FIXME this is run twice with onboardingV1
     }
 
     //TODO remove direct backend access
     Connections {
         enabled: !featureFlagsStore.onboardingV2Enabled
-        target: !featureFlagsStore.onboardingV2Enabled ? startupModule : null
+        target: !featureFlagsStore.onboardingV2Enabled && typeof startupModule !== "undefined" ? startupModule : null
```

Comment:
This fixes the https://github.com/status-im/status-desktop/issues/17222 crash for me

Reply:
isn't it hiding the root cause?

My guess here is that it's crashing when the nim GC will collect the `startupModule` and the fix should come from nim.

We try to set it to an empty `QVariant` just before deleting the `startupModule`. But that `QVariant` is probably collected as well. I don't remember why we're setting an empty `QVariant`, but it's worth trying to set `nil` as `startupModule` context property.

Reply:
This should be the actual fix:
- https://github.com/status-im/status-desktop/pull/17268